server/polar/config.py

```diff
@@ -66,6 +66,9 @@ class Settings(BaseSettings):
     # Stripe webhook secret
     STRIPE_WEBHOOK_SECRET: str = ""
 
+    # Open Collective
+    OPEN_COLLECTIVE_PERSONAL_TOKEN: str | None = None
```

Comment:
Open Collective API can be queried without a token. However, it's limited to 10 req/min. Thus, it's probably better to set up a token of our own to reach 100 req/min.

Ref: https://graphql-docs-v2.opencollective.com/access#authentication

Reply:
+1. I'll fix this before deploying.